packages/babel-preset-default/package.json

```diff
@@ -30,14 +30,12 @@
 	"main": "index.js",
 	"dependencies": {
 		"@babel/core": "^7.13.10",
-		"@babel/plugin-transform-react-jsx": "^7.12.7",
 		"@babel/plugin-transform-runtime": "^7.13.10",
 		"@babel/preset-env": "^7.13.10",
+		"@babel/preset-react": "^7.10.1",
 		"@babel/preset-typescript": "^7.13.0",
 		"@babel/runtime": "^7.13.10",
-		"@wordpress/babel-plugin-import-jsx-pragma": "file:../babel-plugin-import-jsx-pragma",
 		"@wordpress/browserslist-config": "file:../browserslist-config",
-		"@wordpress/element": "file:../element",
```

Comment:
I don't think we need this as a dependency. Peer dependency, probably, but not a dependency.

Reply:
Right, I don't remember why we set them as dependencies. Maybe for developers' convenience because they would need to add `@wordpress/element` as a dependency when using JSX anyway. I guess it's fine to remove, we just should mention it in the changelog.